ts/image-occlusion/shapes/from-cloze.ts

```diff
@@ -89,7 +106,12 @@ export function extractShapesFromRenderedClozes(selector: string): Shape[] {
 
 function extractShapeFromRenderedCloze(cloze: HTMLDivElement): Shape | null {
     const type = cloze.dataset.shape!;
-    if (type !== "rect" && type !== "ellipse" && type !== "polygon") {
+    if (
+        type !== "rect"
+        && type !== "ellipse"
+        && type !== "polygon"
+        && type !== "text"
+    ) {
```

Comment:
Might be worth factoring this out into a separate isValidType(), which could be a [type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)